src/bootstrap/doc.rs

```diff
@@ -478,138 +478,12 @@ impl Step for Std {
             builder.run(&mut cargo);
             builder.cp_r(&my_out, &out);
         };
-        for krate in &["alloc", "core", "std"] {
+        for krate in &["alloc", "core", "std", "proc_macro"] {
```

Comment:
Is it intentional that the `test` crate will no longer be documented?

Reply:
Oops no, that is not intentional.